lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/RootContextLauncher.cs

```diff
@@ -39,17 +40,24 @@ internal sealed class RootContextLauncher
         private ISet<object> _services;
         private ContextRuntime _rootContext;
 
-        public RootContextLauncher(string id, IConfiguration contextConfiguration,
-            Optional<IConfiguration> rootServiceConfig, Optional<IConfiguration> rootTaskConfig, IHeartBeatManager heartbeatManager)
+        public RootContextLauncher(
```

Comment:
Injecting it will prove more troublesome than simply calling the constructor, because then we will need to create `NamedParameters` for the configurations passed in, and it will not be worth the trouble.